packages/core/src/routes/+page.svelte

```diff
@@ -1708,6 +1708,20 @@ const onHoverDelayMsInput = (event: Event) => {
       }}
       cx="max-w-[240px]"
     />
+    <ToastBanner
+      message="Test Message"
+      variant={ToastVariant.Neutral}
+      action={{
+        text: 'Undo',
+        handler: () => {
+          console.log('Clicked action');
+        },
+      }}
+      dismiss={() => {
+        console.log('Clicked close button');
+      }}
+      cx="max-w-[240px]"
```

Comment:
[Nit] As discussed in person, this `max-w-[240px]` conflicts with the banners own class of `max-w-[480px]`. This is a little confusing, so we probably shouldn't add this class in the playground